jasmine-standalone-3.5.0/src/tenPinController.js

```diff
@@ -0,0 +1,50 @@
+
+
```

Comment:
Personally, I'm finding this controller a little confusing, it isn't obvious what elm, prv or two are

Reply:
Agreed - use full, meaningful names